### PR TITLE
KOGITO-2984: Use i18n dictionaries on VS Code extension backend

### PR DESCRIPTION
**Pull request by ljmotta**
## Task
https://issues.redhat.com/browse/KOGITO-2984

## On this PR
 - Added `yarn test:clearCache` on the top level `package.json` which removes all jest caches from the project.
 - Separeted the I18n package into `core` and `react-components`.
 - Created a I18n class.
 - Changed the I18nService to enable more than one subscriber on the locale change.

### CODE REVIEW

**Review by tiagobento — APPROVED** [review]
LGTM

File: packages/chrome-extension/src/app/i18n/setup.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+
+import * as React from "react";
+import { useContext } from "react";
+import { I18nContextType } from "@kogito-tooling/i18n/dist/react-components";
+import { en } from "./locales";
+import { ChromeExtensionI18n } from "./ChromeExtensionI18n";
+import { I18nDefaults, I18nDictionaries } from "@kogito-tooling/i18n/dist/core";
+
+export const chromeExtensionI8nDefaults: I18nDefaults<ChromeExtensionI18n> = {
```
Comment (jomarko) [inline packages/chrome-extension/src/app/i18n/setup.ts]: I think we have a typo here `chromeExtensionI8nDefaults` -> `chromeExtensionI18nDefaults`

**Review by jomarko — COMMENTED** [review]
Just one comment to code inside.

During manual check I tried quickly listed channels:
- HUB :heavy_check_mark: 
- DESKTOP :heavy_check_mark: 
- VS CODE :heavy_check_mark: 
- ONLINE :heavy_check_mark: 
- CHROME :red_circle: 

Screenshot from 2020-08-24 15-06-20

Could someone confirm the error on chrome extension is not caused by this PR?

File: packages/chrome-extension/src/app/i18n/setup.ts (hunk 1 above)
Comment (ljmotta) [reply packages/chrome-extension/src/app/i18n/setup.ts]: Yes! I've fixed it.

**Comment by ljmotta** [comment]
@jomarko Thanks for your review!

About the Chrome Extension, I've built locally using the build:fast and used the chrome-extension artifact generated on this PR build. In both ways, I wasn't able to reproduce the same behavior.

Can you provide more info about how to reproduce this error?

**Review by jomarko — APPROVED** [review]
I retried chrome extension built as part of CI and couldn't reproduce, approving.

**Pull request merged**